### ISSUE TITLE
GSL incorporation

### ISSUE THREAD

**Issue by jgmbenoit** [ntamas/plfit#14]
Have you considered to incorporate plfit in the [GNU Scientific Library (GSL)](https://www.gnu.org/software/gsl/) ?

**Comment by ntamas** [comment]
No, I haven't so far - I'm not that familiar with GSL and I don't know whether it would be a good fit. But I'd be happy to help if someone decides to do the heavy lifting.

**Comment by jgmbenoit** [comment]
NA
